### PR TITLE
Zoom feature

### PR DESCRIPTION
**Pull request by PhilippeLafreniere18**
Simple modifications to allow zoom feature menu in the toast panel.

- Back to menu
- Zoom on feature
- Zoom on features
- Zoom auto

### CODE REVIEW

File: packages/common/src/lib/action/actionbar/actionbar.component.html [hunk 1]
```diff
@@ -51,11 +51,11 @@
 </mat-list>
 
 <div *ngIf="mode === actionbarMode.Overlay">
-  <button
+  <button class="buttonOverlay"
     mat-icon-button
     [matMenuTriggerFor]="actionbarMenu"
     [disabled]="store.view.empty">
-    <mat-icon svgIcon="dots-horizontal"></mat-icon>
+    <mat-icon svgIcon="menu"></mat-icon>
```
Comment (cbourget) [inline packages/common/src/lib/action/actionbar/actionbar.component.html]: Please, make that configurable. The menu icon is not always desirable.

File: packages/common/src/lib/action/actionbar/actionbar-item.component.ts [hunk 2]
```diff
@@ -65,6 +71,13 @@ export class ActionbarItemComponent implements OnInit, OnDestroy {
   set disabled(value: boolean) { this.disabled$.next(value); }
   get disabled(): boolean { return this.disabled$.value; }
 
+  /**
+   * Whether the action is display or not
+   */
+  @Input()
+  set noDisplay(value: boolean) { this.noDisplay$.next(value); }
```
Comment (cbourget) [inline packages/common/src/lib/action/actionbar/actionbar-item.component.ts]: Could be named display instead of noDisplay.

**Comment by pelord** [comment]
It's would be nice if you can also highlight the selected line. At this moment, there is no way to know which line is selected. Maybe a larger line for selected feature?

**Comment by PhilippeLafreniere18** [comment]
> It's would be nice if you can also highlight the selected line. At this moment, there is no way to know which line is selected. Maybe a larger line for selected feature?

Not sure to understand what you are referencing to ? When you are on the list, the "focus'" line is already highlighted ?

**Pull request merged**